.github/workflows/auto_assign_action.yml

```
@@ -10,4 +10,4 @@ jobs:
       - name: 'Auto-assign issue'
         uses: pozil/auto-assign-issue@v1.1.0
         with:
-          assignees: tonyc-phantom,ishans-crest,alexa-phantom,coh-splunk
+          assignees: grokas-splunk,ishans-crest,alexa-phantom,coh-splunk,tapishj-splunk
```

```suggestion
          assignees: grokas-splunk,ishans-crest,phantom-jacob,coh-splunk,tapishj-splunk
```

Alexa, I don't think you should have to continue owning these issues, and I probably should be put on the list